services/net/contentmigration/migrators/PaperMigrator.cs

```diff
@@ -0,0 +1,270 @@
+﻿using LinqKit;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using TNO.API.Areas.Editor.Models.Lookup;
+using TNO.API.Areas.Editor.Models.Product;
+using TNO.API.Areas.Editor.Models.Source;
+using TNO.API.Areas.Editor.Models.Topic;
+using IngestModel = TNO.API.Areas.Services.Models.Ingest.IngestModel;
+using TNO.API.Areas.Services.Models.ContentReference;
+using TNO.Entities;
+using TNO.Kafka.Models;
+using TNO.Services.Actions;
+using TNO.Services.ContentMigration.Config;
+using TNO.Services.ContentMigration.Sources.Oracle;
+using System.Net;
+
+namespace TNO.Services.ContentMigration.Migrators;
+
+/// <summary>
+/// manages Ingestion of TNO 1.0 'Paper'
+/// </summary>
+public class PaperMigrator : IngestAction<PaperMigrationOptions>, IContentMigrator
+{
+    /// <summary>
+    ///
+    /// </summary>
+    public IEnumerable<string> SupportedIngests => new []{
+        "TNO 1.0 - Story Content",
```

Comment:
I'd be better to pull this from the configuration options rather than hardcoded.